res/controllers/common-controller-scripts.js

```diff
@@ -28,6 +28,44 @@ String.prototype.toInt = function() {
     return a;
 };
 
+/**
+ * Discriminates whether an object was created using the `{}` synthax.
+ *
+ * Returns true when was an object was created using the `{}` synthax.
+ * False if the object is an instance of a class like Date or Proxy or an Array.
+ *
+ * Object.isSimpleObject({}) // true
+ * Object.isSimpleObject(null) // false
+ * Object.isSimpleObject(undefined) // false
+ * Object.isSimpleObject(new Date) // false
+ * Object.isSimpleObject(new (class {})()) // false
+ * @param {any} obj Object to test
+ * @returns {boolean} true if obj was created using the `{}` synthax, false otherwise
+ */
+Object.isSimpleObject = function(obj) {
+    return obj !== null && typeof obj === "object" && obj.constructor.name === "Object";
+};
```

Comment:
fyi `Object.isSimpleObject(new Object()) === true`, probably doesn't matter in terms of behavior, but the comment/naming is a little misleading.

Reply:
True.